webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js

```diff
@@ -39,6 +39,8 @@ class SakaiRubricAssociation extends RubricsElement {
       associateValue: { attribute: "associate-value", type: Number },
       fineTunePoints: { attribute: "fine-tune-points", type: String },
       hideStudentPreview: { attribute: "hide-student-preview", type: String },
+      studentSelfReport: { attribute: "student-self-report", type: String },
+      showSelfReportCheck: { attribute: "show-self-report-check", type: String },
```

Comment:
Both should be Boolean

Reply:
ok.
but studentSelfReport is a string ->"Allow students to self-report their work"